### PR TITLE
DSD-832: Removes the .nypl-ds css class rule

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-832](https://jira.nypl.org/browse/DSD-832)

## This PR does the following:

- Removes the `.nypl-ds` CSS class style rule and its references.
- Removes the base styles under the `nypl-ds` class to the global Chakra theme JS object.

## How has this been tested?

Storybook and tests. Checked components that were updated and that had its focus styles moved from the SCSS file to the JS theme object. We should eventually update and standardize how the focus should look like and what components use it. For now, it's the same but just moved from one system to another.

## Accessibility concerns or updates

- N/A, the focus style is the same.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr895-tqsjkorfsb7w9arjd30txc1bjqnfbokr.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6227ccfbb1d827ef08a04de1

File: src/theme/components/global.ts [hunk 1]
```diff
@@ -4,13 +4,12 @@ export { wrapperStyles } from "./globalMixins";
 /** Reusable component styles. */
 
 // Used in `Select` and `TextInput`.
-const activeFocus = {
-  border: "1px solid",
-  borderColor: "ui.focus",
+const activeFocus = (darkMode = false) => ({
```
Comment (bigfishdesign13) [inline src/theme/components/global.ts:7]: Prepping for what we discussed this morning?

**Review by bigfishdesign13 — APPROVED** [review]
This looks good.  It’s amazing that we are finally here.

File: src/theme/components/global.ts (hunk 1 above)
Comment (EdwinGuzman) [reply src/theme/components/global.ts:7]: Actually, this is based on the current SCSS `focus-outline` mixin. I didn't want to deviate from what was existing, just move it over from scss to css-in-js.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr895-tqsjkorfsb7w9arjd30txc1bjqnfbokr.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6227ccfbb1d827ef08a04de1

**Pull request merged**